Tools/CISupport/ews-build/steps.py

```diff
@@ -1496,11 +1500,11 @@ def propertiesToPassToTriggers(self, patch=True, pull_request=False):
             ]
         if self.triggers:
             property_names.append('triggers')
-        if self.include_revision and pull_request:
-            properties_to_pass['ews_revision'] = properties.Property('got_revision')
 
         properties_to_pass = {prop: properties.Property(prop) for prop in property_names}
         properties_to_pass['retry_count'] = properties.Property('retry_count', default=0)
+        if self.include_revision and pull_request:
```

Comment:
this 'pull_request' check was incorrect in previous commit (as noted in https://github.com/WebKit/WebKit/pull/93#issuecomment-1021732074).